macrel/macrel_features.py

```diff
@@ -0,0 +1,184 @@
+import logging
+import numpy as np
+from .database import eisenberg, instability, _aa_groups
+from .database import pos_pks, neg_pks, boman_scale
+from collections import Counter
+
+class MacrelFeatures:
+
+    '''
+    Functions were adapted from:
+
+      -- ModlAMP(https://github.com/alexarnimueller/modlAMP)
+      -- Peptides R Package (https://github.com/dosorio/Peptides), and
```

Comment:
Does this make the code GPL? How much adaptation was there?

Reply:
modlAMP is BSD3 which is free to share/modify/distribute just acknowledging to the authors. The code itself changed, however, some functions had very few modifications, and there are some functions that they did not have, such as hydrophobicity. I am still wondering if we should include the synthesis/solubility rules there and change the output a bit, giving also those info, what do you think?